src/app/utils/custom-errors.ts

```diff
@@ -0,0 +1,23 @@
+export class ConflictError extends Error {
+  /**
+   * Custom name of this sub-class of Error
+   */
```

Comment:
can we take the opportunity to JSDoc these errors properly?

Reply:
done